docs/development/how-to-file-issue.md

```diff
@@ -0,0 +1,44 @@
+# How to file an Issue
```

Comment:
I think somewhere on this page it's good to mention something along the lines of not being intimidated by some of the information we're looking to gather. At the end of the day, we're interested in things being opened, and can work to gather the actual info. But I don't want people to avoid filing issues just because they aren't sure about a few of the requirements, regardless of the type of issue they're filling.